goforit.go

```diff
@@ -466,11 +493,10 @@ func (g *goforit) ReportCounts(callback func(name string, total, enabled uint64,
 	}
 
 	duration := time.Now().Sub(start)
-	if g.stats != nil {
-		_ = g.stats.Gauge(reportCountsScannedMetricName, float64(scanned), nil, 1.0)
-		_ = g.stats.Gauge(reportCountsReportedMetricName, float64(reported), nil, 1.0)
-		_ = g.stats.TimeInMilliseconds(reportCountsDurationMetricName, duration.Seconds()*1000, nil, 1.0)
-	}
+	stats := g.getStats()
```

Comment:
this will now never return a `nil` stats instance - it will return a noop instance if the backing instance is nil